config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -344,6 +342,58 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  # clone of pull-perf-tests-clusterloader2 with enabled node killer and using only load test
+  - name: pull-perf-tests-clusterloader2-nodekiller-manual
```

Comment:
Make it explicit that it's an experimental thing..

pull-perf-tests-clusterloader2-nodekiller-experimental?

Reply:
Done